### DISCUSSION TITLE
Padronização contatos Python Sul

### DISCUSSION THREAD

**Discussion by talitarossari** [apyb/comunidade#101] [Geral]
Opa pessoal, tudo certo?
Irei puxar a Python Sul de 2024, e durante o processo de recuperar as senhas/repositórios notei que a python sul esta despadronizada.
Nosso email é `pythonsul@gmail.com`, enquanto outras regionais utilizam o @python.org.br.
É possível criar um email `sul@python.org.br` para utilizarmos?

Além disso, vejo que é uma oportunidade de centralizarmos algum tipo de URL para acessar os eventos regionais.
É possível criar um dominio/subdominio como `2024.sul.python.org.br` para utilizarmos?
Verifiquei que hoje o caipyra esta nessa estrutura hoje, e ela faz sentido, assim só mudo a parte do evento e acesso todos os eventos regionais :)

Agradeço desde já <3
Talita Rossari.

**Comment by rougeth** [comment]
Oi @talitarossari! :)

A gente não fornece emails para nenhum evento regional, apenas Python Brasil pelo eventos@.

> É possível criar um dominio/subdominio como 2024.sul.python.org.br para utilizarmos?

Isso sim! :D Vocês já tem um endereço para apontarmos nesse subdomínio?

**Comment by talitarossari** [comment]
Opa @rougeth, tudo certo?

Sobre o email, a Python nordeste tem um email com @python.org.br... por isso minha pergunta. O endereço está no rodapé do site do evento: 2023.pythonnordeste.org

Sobre o endereço, @gabrielecker, pode me ajudar com isso?

**Reply by rougeth** [reply]
> Sobre o email, a Python nordeste tem um email com @python.org.br... por isso minha pergunta. O endereço está no rodapé do site do evento: 2023.pythonnordeste.org

Tinha me esquecido sobre isso. Vou ver isso na próxima semana e aviso aqui.

**Reply by talitarossari** [reply]
Algum retorno sobre isso? :3

**Reply by naanadr** [reply]
@rougeth vi aqui que o e-mail já foi criado. @talitarossari você já está com a senha do e-mail?

**Comment by gabrielecker** [comment]
Oi @rougeth, por enquanto estamos usando o pythonsul.dev, mas pelo que eu entendi a ideia é deixar como o principal esse da apyb, certo?
Vou tentar descobrir quem está mantendo o pythonsul.dev inclusive, para avaliar se podemos encerrá-lo

**Reply by rougeth** [reply]
> como o principal esse da apyb, certo?

Não não, a APyB não quer padronizar nem centralizar como os eventos se organizam. Podemos fornecer subdomínios no endereço python.org.br caso os eventos queiram. Podemos também assumir a manutenção e os custo dos domínios customizados dos eventos que já são estabelecidos, como a Python Nordeste.

Mas isso tem que partir da organização do evento, se vocês quiserem usar pythonsul.dev, show, se quiserem usar sul.python.org.br, show também :).

**Comment by Perceu** [comment]
Boa tarde! turma tava falando com o @gabrielecker sobre esses dominios e esta um pouco contrangedor termos perdido o pythonsul.org para conteudos adultos (não visitem), resolvi ver a disponibilidades dos .com.br e com isso acabei tentando e comprando todos os python[macroregião].com.br

pythoncentrooeste.com.br
pythonsul.com.br
pythonnorte.com.br
pythonnordeste.com.br
pythonsudeste.com.br

todos estão pagos por 1 ano não me importo em assumir o custo desses dominios se for bom para a comunidade, mas acho inrresponsavel da minha parte isso ficar vinculado a minha conta do registrobr, visto isso me disponibilizo a seder esses dominios para a APYB que vcs acham ?

se não for do interesse da associação eu vou tentar manter indexado os eventos de cada macro região nesses dominios, talvez só de ajuda mas isso e papo pra outra tread

**Reply by rougeth** [reply]
> esta um pouco contrangedor termos perdido o pythonsul.org 

Será que existe alguma forma de tentarmos recuperar o domínio? Quem estava cuidando dele?

> visto isso me disponibilizo a seder esses dominios para a APYB que vcs acham ?

Tem alguns problemas com esses domínios. Python Nordeste usa o mesmo domínio .org a vários anos, acho que não faria muito sentido mudar para .com.br. Comunidade do centro-oeste usa o nome Python Cerrado, daí o domínio também é diferente.

Acho que manter só como redirecionador não me parece necessário, os links reversos sempre apontarão para o domínio correto.

**Reply by Perceu** [reply]
por mim zero problemas, não sabia o esquema do cerrado 😋, foi por impulso para evitar constrangimentos, e aproveitar oportunidade, todos eles ficam disponiveis comigo por um ano a comunidade que precisar eu repasso senão pretendo deixa-los vencer.  Sem fazer uso para tambem não dar problema com os links reversos então.

**Reply by gabrielecker** [reply]
bom dia gente, pensamos em fazer dessa forma pois assim mantemos nosso endereço mais "intuitivo", a exemplo do caipyra, python nordeste, etc, além de que fica mais fácil a rastreabilidade dos responsáveis pelo domínio, visto que estará com a apyb, e também não precisamos nos preocupar em renovar o domínio, pois tivemos um caso constrangedor com o domínio da pythonsul.org
não necessariamente queremos impor isso para todos, mas seria uma boa ideia apenas sugerir para os eventos regionais que queiram essa possibilidade de centralizar essa responsabilidade
sobre os domínios que o perceu comentou, se já estão comprados poderíam utilizá-los apenas para redirecionar para o correto, similar ao que as empresas fazem (ig.com, fb.com, etc), não?

**Reply by naanadr** [reply]
@gabrielecker não entendi, então vocês vão utilizar esse ano o dominio `pythonsul.com.br` ?

>sobre os domínios que o perceu comentou, se já estão comprados poderíam utilizá-los apenas para redirecionar para o correto, similar ao que as empresas fazem (ig.com, fb.com, etc), não?

Concordo com você @gabrielecker . @Perceu o redirect tem que ser do seu lado?

**Comment by talitarossari** [comment]
Opa @naanadr já esta tudo certo, agradeço o apoio de todes <3
vou fechar o PR
espero ver geral na pysul <3
